modules/API/pages/built-in-endpoints.adoc

```diff
@@ -3384,6 +3384,168 @@ Response::
+      "name": "louvain",
+      "params": {
+        "file_path": {
+          "default": "",
+          "type": "STRING",
+          "min_count": 0
+        },
+        "print_info": {
+          "default": "false",
+          "type": "BOOL",
+          "min_count": 0
+        },
+        "wt_attr": {
+          "default": "weight",
+          "type": "STRING",
+          "min_count": 0
+        },
+        "result_attr": {
+          "default": "cid",
+          "type": "STRING",
+          "min_count": 0
+        },
+        "max_iter": {
+          "default": "10l",
+          "type": "INT64",
+          "min_count": 0
+        },
+        "v_type": {
+          "max_count": 2147483647,
+          "type": "STRING",
+          "min_count": 0
+        },
+        "e_type": {
+          "max_count": 2147483647,
+          "type": "STRING",
+          "min_count": 0
+        }
+      }
+    }
+  ]
+}
+----
+--
+====
+==== Parameters
+
+|===
+| Name | Required | Description
+
+| `functionName`
+| Yes
+| The template name that will be returned in the response
+
+| `isRegularExpression`
+| No
+| Enables regular expression searching for the function name. For example, `gds.*` returns all queries that begin with `gds`.
```

Comment:
please fulfill the usage of `isRegularExpression` if @yimingpantg has explicited it